pyaedt/generic/touchstone_parser.py

```diff
@@ -545,7 +545,7 @@ def check_touchstone_files(folder="", passivity=True, causality=True):
                     causality_check = float(msg_log.split("Maximum causality error: ")[-1].split("for entry")[0])
                     if not causality_check == 0.0:
                         is_causal = False
-                except:
+                except Exception:
                     is_causal = False
                     raise Exception("Failed evaluating causality value")
```

Comment:
```suggestion
                    raise Exception("Failed evaluating causality value.")
```
Messages should have verb (Failed) and also concluding punctuation.